Prepare for 0.7.0 release (#61)
scrapy-plugins/scrapy-zyte-api: CHANGES.rst

Changes
=======

0.7.0 (TBD)
-----------

> To upgrade, you need to set the following in your Scrapy project settings:

Reviewer: - It is not technically necessary
- These Scrapy settings can be defined at the spider level

```suggestion
When upgrading, you should set the following in your Scrapy settings:
```

.. code-block:: python

  DOWNLOADER_MIDDLEWARES = {
      "scrapy_zyte_api.ScrapyZyteAPIDownloaderMiddleware": 1000,
  }
  # only applicable for Scrapy 2.7+
  REQUEST_FINGERPRINTER_CLASS = "scrapy_zyte_api.ScrapyZyteAPIRequestFingerprinter"

* Fixes the issue where scrapy-zyte-api is slow when Scrapy Cloud has Autothrottle
  Addon enabled. The new ``ScrapyZyteAPIDownloaderMiddleware`` fixes this.

* It now supports Scrapy 2.7's new ``REQUEST_FINGERPRINTER_CLASS`` which ensures
  that Zyte API requests are properly fingerprinted. This addresses the issue

>   where Scrapy marks POST requests as duplicate if it points to the same URL

Reviewer 2: "if they point" maybe?
